### PR TITLE
Fix Generated Error Types in JSDoc

### PR DESCRIPTION
**Pull request by samialfhaily**
## Before this PR
The documented error types in JSDoc did not reference actual TS types.

## After this PR
==COMMIT_MSG==
Change documented error types in JSDoc to reference the actual generated error types by Conjure
==COMMIT_MSG==

---

Relate PR: https://github.com/palantir/conjure-typescript/pull/294

### CODE REVIEW

File: src/commands/generate/serviceGenerator.ts [hunk 1]
```diff
@@ -137,6 +137,21 @@ export function generateService(
             scope: Scope.Public,
             docs: docs != null ? [docs] : undefined,
         });
+
+        endpointDefinition.errors?.forEach(error => {
+            imports.push(
```
Comment (PhoebeSzmucer) [inline src/commands/generate/serviceGenerator.ts]: Is it possible that we get multiple imports for the same error in the same file?

Comment (samialfhaily) [reply src/commands/generate/serviceGenerator.ts]: No, imports are deduped before that are added. I also added a test to make sure that we don't do duplicate imports.

File: scripts/run-integration-tests.sh [hunk 2]
```diff
@@ -5,4 +5,8 @@ set -euo pipefail
 ./build/downloads/bin/conjure-verification-server ./build/resources/verification-server-test-cases.json ./build/resources/verification-server-api.conjure.json &
 SERVER_PID=$!
 yarn karma start --single-run --browsers ChromeHeadless karma.conf.js
-kill -kill ${SERVER_PID}
\ No newline at end of file
+
+if ps -p ${SERVER_PID} > /dev/null
+then
+    kill -kill ${SERVER_PID}
+fi
```
Comment (samialfhaily) [inline scripts/run-integration-tests.sh:12]: Added this to make sure that the server process is still running before we kill it. Otherwise, the script would terminate with an error code if the server process was not not running.

Comment (samialfhaily) [inline scripts/run-integration-tests.sh:12]: Added this to make sure that the server process is still running before we kill it. Otherwise, the script would terminate with an error code if the server process was not not running.

**Review by PhoebeSzmucer — APPROVED** [review]
